src/LinuxTracingIntegrationTests/IntegrationTestPuppet.h

```diff
@@ -30,6 +30,8 @@ struct IntegrationTestPuppetConstants {
 
   constexpr static uint64_t kFrameCount = 60;
 
+  constexpr static uint64_t kRssIncreaseB = 100ULL * 1024 * 1024;
```

Comment:
Nit: Is this random? If not consider adding a comment.

Reply:
Added comment "Large enough to be well measurable."